thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -859,10 +860,14 @@ class CDistributorBase : implements IHashDistributor, implements IExceptionHandl
                     }
                     if (aborted)
                         break;
-                    const void *row = input->ungroupedNextRow();
+                    const void *row;
+                    MEASURE_CYCLES_ATOMIC(lookAheadCycles, owner.activity->queryTimeActivities(),
```

Comment:
not a fan of macros :)
I doubt the complexity added to check enabled is worth. I vote for keeping it simple (and avoiding the macro), e.g.:

```
                    CCycleTimer rowTimer;
                    const void *row = input->ungroupedNextRow();
                    lookAheadCycles.fastAdd(rowTimer.elapsedCycles());
```